torch/onnx/symbolic_opset13.py

```diff
@@ -417,3 +418,43 @@ def diagonal(g, self, offset, dim1, dim2):
     final_overrun_ = zeros(else_block, gather_shape, 6, None, None)
     _add_output_to_block(else_block, final_overrun_)
     return if_op
+
+class Quantized:
+    """
+    https://github.com/pytorch/pytorch/wiki/PyTorch-ONNX-exporter#quantized-model-export
+    """
+    domain = "quantized"
+
+    @staticmethod
```

Comment:
non-blocking: Is there a way we don't need to staticmethods, or are there special considerations?

https://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators and https://mail.python.org/pipermail/python-ideas/2016-July/041189.html from Guido himself.

Reply:
We could switch to using a module rather than a class for each domain.
But neither of those links explains why we should not use static methods.
Feel free to send a PR to change to modules if you can provide a good justification.

Reply:
The main argument I know of is that we usually write static methods in a class just for a name space, whereas the Pythonic way of name spacing is via modules. I assume things can be different if we wanted to implement some kind of protocols with the class, overwriting the same signatures for example. In that case the class will be a reference when used and its name not spelled out in the caller. I wonder if that would add obscurity to the code and if that is something we were trying to do with this class.